data/items.py

```diff
@@ -187,6 +188,20 @@ def moogle_starting_equipment(self):
             self.characters.characters[index].init_body = random.choice(tiers[Item.ARMOR][1])
             self.characters.characters[index].init_head = random.choice(tiers[Item.HELMET][1])
 
+    def moogle_curse_description(self):
+        # If using -no-random-encounters, the Moogle Charm becomes the Moogle Curse
+        # (enables random encounters on world map).  Update the name and description.
+        moogle_charm = self.items[name_id["Moogle Charm"]]
+        moogle_charm.name = "Moogle Curse"
+
+        descr_offset = self.rom.get_bytes(0x2d7aa0 + moogle_charm.id * 2, 2)
+        offset = descr_offset[1] * 0x100 + descr_offset[0]
+
+        # new_descr = "No random enemy encounters<end>"
+        new_descr = "Draw monsters on world map<end>"
+        name_bytes = text.get_bytes(new_descr, text.TEXT2)
+        self.rom.set_bytes(0x2d6400 + offset, name_bytes)
```

Comment:
Please structure the setting of an item description like this (make a desc_data array in items.py and set the description in item.py):

https://github.com/ff6wc/WorldsCollide/pull/28/files#diff-5991f3562b7b58a2c5931c6a2897859699dea4634f1e33cd7511655c86c1165d